File: July 2024 GFG SOLUTION/July-28.cpp
//{ Driver Code Starts
#include <bits/stdc++.h>
using namespace std;


// } Driver Code Ends
// User function template for C++
class Solution {
  public:
    string removeDups(string str) {
        int fre[26] = {0};
        string result = ""; 
        
        for (char c : str) {
            if (fre[c - 'a'] == 0) { 
                result += c; 
                fre[c - 'a'] = 1; 
            }
        }
        
        return result;
    }
};


//{ Driver Code Starts.

int main() {

    ios_base::sync_with_stdio(0);
    cin.tie(NULL);
    cout.tie(NULL);

    int t;
    cin >> t;
    while (t--) {
        string s;
        cin >> s;

        Solution ob;
        cout << ob.removeDups(s) << "\n";
    }

    return 0;
}
// } Driver Code Ends

File: July 2024 GFG SOLUTION/July-28.py
#User function Template for python3
class Solution:
    def removeDups(self, str):
        fre = [0] * 26 
        result = [] 
        
        for c in str:
            if fre[ord(c) - ord('a')] == 0:
                result.append(c) 
                fre[ord(c) - ord('a')] = 1 
        
        return ''.join(result)


#{ 
 # Driver Code Starts
#Initial Template for Python 3

if __name__ == '__main__':
    T = int(input())
    for i in range(T):
        s = input()

        ob = Solution()
        answer = ob.removeDups(s)

        print(answer)

# } Driver Code Ends
